cmake/Modules/EttercapLibCheck.cmake

```diff
@@ -117,22 +117,18 @@ endif(HAVE_ICONV)
 
 # LTDL
```

Comment:
How about systems where libdl is not installed? I guess this would result in a build failure. This was where I was aiming at. 

Or we have to make the availability of libdl mandatory.